site/public/css/authentication.css

```diff
@@ -51,3 +51,32 @@
 #saml-login {
     color: var(--btn-text-white);
 }
+
+.hidden-helper {
+    display: none;
+    padding: 10px;
+    font-size: small;
+}
+
+.display-box input {
+    display: block;
+    width: 100%;
+}
+
+.requirements-button {
+    display: inline-block;
+    font-weight: 400;
+    color: var(--standard-deep-dark-grey);
+    text-align: center;
+    vertical-align: middle;
+    background-color: transparent;
+    border: 1px solid;
+    padding: 0.3rem;
+    font-size: x-small;
+    line-height: 1;
+    border-radius: 0.25rem;
+}
+
+[data-theme="dark"].requirements-button {
+    color: var(--text-white);
```

Comment:
Is there a color which changes automatically based on the theme you can choose instead?